### DISCUSSION TITLE
Not able to login with Okta

### DISCUSSION THREAD

**Discussion by ahmedabrar914** [netbox-community/netbox#17227] [Help Wanted!]
I have configured in the configuration.py file with the following details. I followed the steps from  https://www.oasys.net/posts/okta-sso-with-netbox/ 

REMOTE_AUTH_BACKEND = 'social_core.backends.okta_openidconnect.OktaOpenIdConnect'
SOCIAL_AUTH_OKTA_OPENIDCONNECT_KEY= 'XXXXXX
SOCIAL_AUTH_OKTA_OPENIDCONNECT_SECRET= 'XXXXXX'
SOCIAL_AUTH_OKTA_OPENIDCONNECT_API_URL= 'https://xxxxxxxxxx/oauth2/'

SOCIAL_AUTH_PIPELINE = [
    "social_core.pipeline.social_auth.social_details",
    "social_core.pipeline.social_auth.social_uid",
    "social_core.pipeline.social_auth.auth_allowed",
    "social_core.pipeline.social_auth.social_user",
    "social_core.pipeline.social_auth.associate_by_email",
    "social_core.pipeline.user.create_user",
    "social_core.pipeline.social_auth.associate_user",
    "social_core.pipeline.social_auth.load_extra_data",
    "social_core.pipeline.user.user_details"
]
REMOTE_AUTH_HEADER = 'HTTP_REMOTE_USER'
REMOTE_AUTH_AUTO_CREATE_USER = True

I am getting the error as page not found. Please help me to resolve this issue

<img width="776" alt="Screenshot 2024-08-21 at 3 47 18 PM" src="https://github.com/user-attachments/assets/72d25d74-0287-4aea-8960-86902046e423">

<img width="776" alt="Screenshot 2024-08-21 at 3 46 07 PM" src="https://github.com/user-attachments/assets/c79384b0-26f0-4182-9da8-cb5150db961d">

**Comment by candlerb** [comment]
Please don't ping me personally. I don't use Okta and I don't use python-social-auth.

You are following a random document you found on the Internet, and I don't know if that's correct or not; there is [official documentation](https://netboxlabs.com/docs/netbox/en/stable/administration/authentication/okta/) for integrating Netbox with Okta.

I do notice though that both documents say the sign-in redirect URL you should configure in Okta is `/oauth/complete/okta-openidconnect/`, not `/oauth/login/okta-openidconnect/`.
